ejb/src/main/java/org/wildfly/httpclient/ejb/HttpEJBReceiver.java

```diff
@@ -283,7 +283,7 @@ protected SessionID createSession(final EJBReceiverSessionCreationContext receiv
         CompletableFuture<SessionID> result = new CompletableFuture<>();
 
         RequestBuilder builder = new RequestBuilder()
-                .setRequestType(RequestType.CREATE_SESSION)
+                .setRequestType(RequestType.OPEN)
```

Comment:
Because there is no open operation in the EJB spec, I am not very fond of this renaming. CREATE_SESSION implies that we are creating the session, which is what we actually are doing here. OPEN is confusing. We use open to refer to transactions, resources, connections, but to me it does not relate to the create session operation.